lib/MD5HashStream.cs

```diff
@@ -50,6 +50,8 @@ internal class MD5HashStream : IDisposable
         /// </summary>
         private long md5hashOffset;
 
+        private bool canSeek = true;
```

Comment:
Minor: Would better add comments as it might be only for non-seekable, non-resumable case, in case of confusion.

Reply:
Thanks for your comments. Will add more help info in next PR.